scripts/recompute_snqi_weights.py

```
@@ -0,0 +1,488 @@
+            "strategy": strategy,
+            "weights": weights,
+            "statistics": stats,
+            "normalization_strategy": "median_p95",
+        }
+
+        if strategy == "pareto":
+            result["pareto_alternatives"] = pareto_samples[:5]  # Include top 5 alternatives
+
+        return result
+
+    def compare_normalization_strategies(self, base_weights: Dict[str, float]) -> Dict[str, Any]:
```

Pylint (3.3.7)

[refactor] Too many local variables (16/15)

(R0914)